elements/rh-tabs/rh-tab-panel.css

```diff
@@ -0,0 +1,20 @@
+#rhds-container {
+  display: block;
+  padding: var(--rh-space-2xl, 32px);
```

Comment:
```suggestion
  padding: var(--rh-space-3xl, 48px);
```

Reply:
This is part of the conversation I'm going to have with @marionnegp later today.